ui/shared/tabs/failureSummary/SuggestionsListItem.jsx

```diff
@@ -110,7 +119,25 @@ export default class SuggestionsListItem extends React.Component {
           >
             <FontAwesomeIcon icon={faBug} title="File bug" />
           </Button>
-          <span>{suggestion.search}</span>
+          <span>{suggestion.search} </span>
+          <Clipboard description="error" text={suggestion.search} />
+          <a
+            href={getLogViewerUrl(
+              selectedJob.id,
+              repoName,
+              suggestion.line_number + 1,
+            )}
+            target="_blank"
+            rel="noopener noreferrer"
+            title="Go to log viewer"
+          >
+            <img
```

Comment:
We use the `FontAwesomeIcon` library component for these icons (see the Clipboard component for reference).

Reply:
The `logviewerIcon` is a special, custom-made icon.  I don't believe FontAwesome has anything like it.

Reply:
oh! That makes sense :)